src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -533,6 +530,10 @@ namespace winrt::TerminalApp::implementation
         void _SelectionMenuOpened(const IInspectable& sender, const IInspectable& args);
         void _PopulateContextMenu(const IInspectable& sender, const bool withSelection);
         winrt::Windows::UI::Xaml::Controls::MenuFlyout _CreateRunAsAdminFlyout(int profileIndex);
+
+        winrt::Microsoft::Terminal::Control::TermControl _senderOrActiveControl(const winrt::Windows::Foundation::IInspectable& sender);
+        winrt::com_ptr<TerminalTab> _senderOrFocusedTab(const IInspectable& sender);
```

Comment:
For the future: technically, we might want the concept of a "stack" of senders. A message coming from a control has an associated pane and tab _and eventually, window_. Those things might matter to different recipients, or different action types.